src/project/project.controller.ts

```diff
@@ -67,6 +67,52 @@ export class ProjectController {
     }
   }
 
+  @Get('/all')
+  @ApiQuery({ name: 'take', required: false })
+  @ApiQuery({ name: 'skip', required: false })
+  @UseGuards(RolesGuard)
+  @Roles(UserType.ADMIN)
+  @ApiResponse({ status: 200, type: Project })
+  async getAllProjects(@Query('take') take: number | undefined, @Query('skip') skip: number | undefined): Promise<Project | object> {
+    this.logger.log({take, skip }, 'GET /project/:id')
+
+    const where = Object()
+    return await this.projectService.paginate({take, skip}, where)
+  }
+
+
+  @Get('/uuid/:id')
```

Comment:
```suggestion
  @Get('/user/:id')
```

тут лучше вместо uuid вписать user что-бы было понятнее